ui-tests/test/contents.spec.ts

```diff
@@ -100,6 +100,11 @@ test.describe('Contents Tests', () => {
     expect(output).toBeTruthy();
     expect(output![0]).toBe('4');
 
+    await page.notebook.close();
+    // wait for the state to be saved
+    // TODO: find a better way than waiting?
```

Comment:
Left a comment to check if there could be a way to avoid the `page.waitForTimeout` to get around the lab debouncer.